LibraBFT/Prelude.agda

```diff
@@ -426,6 +433,14 @@ module LibraBFT.Prelude where
   forM_      []  _ = return unit
   forM_ (x ∷ xs) f = f x >> forM_ xs f
 
+  -- NOTE: this is call forM (no prime) in Haskell.  But because of above forM_ that does not work.
```

Comment:
s/call/called/

Reply:
Can't parse the second sentence.

Reply:
I just updated this with a different solution.
Please review.

Reply:
Looks good.